### DISCUSSION TITLE
personal-development-farvardin-1403/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#40] [Announcements]
# personal-development-farvardin-1403/

ماه فروردین هم به آخر رسید و طبق قرارم با خودم، گزارش این ماه رو منتشر می‌کنم.
در کل ماه خیلی خوبی بود و با این که نصف ماه درگیر اسباب‌کشی و تعمیر خونه و کارهای جانبی بودم، برام رضایت‌بخش بود.
البته از نظر زمانی تفاوت چندانی نکرده و باز هم وقت نمی‌کنم به بعضی‌ها ایمیل‌ها و پیام‌هایی که می‌گیرم جواب بدم یا کار عجیب غریبی بکنم ولی همین که درگیر ترافیک نیستم برام خوبه.

https://aprd.ir/personal-development-farvardin-1403/

**Comment by shayan15sa** [comment]
مثل همیشه عالی
بعد برای ویندوز و ***ان‌شاءالله*** بعداً برای لینوکس. 😂

**Reply by pourmand1376** [reply]
سلام شایان جان. 

بله همین‌طوره و من همیشه حس عقب‌افتادن داشتم که نمی‌تونستم از یک سری از ابزارها استفاده کنم. مثلاً ترمینال Warp رو نگاه کنید. مدت‌هاست برای مک اومده و نسخه لینوکسش هم بعداً اومد ولی نسخه ویندوز نداره. 

یا مثلاً مرورگر Arc که اخیراً خیلی دارم باهاش کار می‌کنم فقط نسخه مک داره و ویندوزش هنوز بتا هست. لینوکس هم که اصلاً نداره. 

حتی اگر دقت کنی نسخه تلگرام دسکتاپ تو مک خیلی بهتر از نسخه ویندوز و لینوکس هست. کلی فیچر داره که اونجا هنوز اضافه‌اش نکرده. 

شاید خیلی‌ها براشون مهم نباشه ولی من به شخصه خوره اینجور چیزها هستم و تو ذوقم می‌خوره وقتی نمی‌تونم به خاطر سیستم‌عامل چیزی رو نصب کنم. 

پی‌نوشت: دیدم وبلاگت رو با هوگو درست کردی و برام جالب بود. کلاً از این تم خوشم میاد.

**Reply by shayan15sa** [reply]
نه دقیقاً، حق با توئه. من اولین بار که با این مشکل رو‌به‌رو شد رو دقیقاً یادم هست. 

3 سال پیش تقریباً کلاس نهم بود (الان یازدهم هستم). کرونا بود و مدارس هم آنلاین شده بود. یه مقداری وقت اضافه داشتم. همینجوری توی یوتوب با جادی آشنا شدم. تقریباً بعد دیدن 3 تا از ویدیو هاش بعد از اینکه فهمیدم لینکوس چیه، به سرعت توزیع ابنوتو رو  dual boot نصب کردم. اصلاً یه حال خاصی بود برای اولین بار توی ترمینال چندتا دستور نوشتم.

یه مدت که استفاده کردم، خواستم واتساپ رو نصب بکنم (اون موقع هنوز فیلتر نشده بود). که فهمیدم نسخه لینوکس نداره که البته همون نسخه وبش بود الکترون. (اتفاقاً یادم که کرونا هم گرفته بود وقتی که با این مشکل رو‌به‌رو شدم)

اما البته همین باعث شد که من نرم‌افزار آزاد بیشتر آشنا و علاقه‌مند بشم. برای مثال از GIMP و Kdelive و هزار تا جایگزین دیگه که آزاد بودن آشنا شدم.

**Comment by mhrn83** [comment]
<b>سلام </b>
فقط می خواستم بگم وقت هایی که حال هیچ چیز رو ندارم میام پست ها رو می خونم و برام جالبه

**Reply by pourmand1376** [reply]
سلام مهران جان.

خوشحالم که چیزهایی که می‌نویسم برات جالبه. راستش خودمم کیف می‌کنم از این که یه جایی دارم که می‌تونم راجع به دغدغه‌هام و چیزهایی که تو ذهنم می‌گذره، صحبت کنم.
